### ISSUE TITLE
Export Permissions for Public Roams

### ISSUE THREAD

**Issue by jacklaing** [Roam-Research/issues#333]
**Problem**

If you have a publicly viewable Roam database, even with read-only permissions, anyone can export your entire database, even if they're not signed in.

**Solution**

Add an option in the share settings to decide whether exporting should be available to read-only users, editors, etc.

**Comment by classicrob** [comment]
Yes please! I don't want people to be able to download my entire public database. If they want to go through all of the effort to copy paste everything and resolve all of the block level transclusions and such, be my guest, but I don't want downloading the database to be easy unless I say that I want them to be able to do it.

Generally I want more advanced permissioning, with something like comment-only permissioning enabled.

**Comment by jacklaing** [comment]
My thoughts exactly!

While we're on the subject of advanced permissioning, it would also be very cool to expose the "Add Version" action to read-only users, while still preventing them from editing the main version, for open-source git-style collaboration https://github.com/Roam-Research/issues/issues/73

**Comment by malcolmocean** [comment]
Yes, I would like this

**Comment by chinarut** [comment]
agreed.  and this is especially important now that we have a book club.
